jujuclient/file.go

```diff
@@ -494,11 +495,42 @@ func (s *store) CurrentModel(controllerName string) (string, error) {
 			controllerName,
 		)
 	}
+
+	var controller bool
```

Comment:
This var is a little vague. Given w ehave `controllerName` in this scope as well, this made it a little difficult to parse. Perhaps `hasControllerModel` or smth instead?